File: py/bprint.py
import time

def bprint(text):
    text_length = len(text)
    border_length = text_length + 4

    for i in range(border_length):
        print("*", end="", flush=True)
        time.sleep(0.055)
    print("")

    print("* ", end="")
    for i in range(text_length):
        print(text[i], end="", flush=True)
        time.sleep(0.055)
    print(" *")

    for i in range(border_length):
        print("*", end="", flush=True)
        time.sleep(0.055)
    print("")

File: c/src/bprint.c
#include <stdio.h>
#include <unistd.h>
#include <string.h>

void bprint(char *text) {
    int text_length = strlen(text);
    int border_length = text_length + 4;

    for (int i = 0; i < border_length; i++) {
        putchar('*');
        fflush(stdout);
        usleep(19500);
    }
    printf("\n");

    printf("* ");
    for (int i = 0; i < text_length; i++) {
        putchar(text[i]);
        fflush(stdout);
        usleep(19500);
    }
    printf(" *\n");

    for (int i = 0; i < border_length; i++) {
        putchar('*');
        fflush(stdout);
        usleep(19500);
    }
    printf("\n");
}
